pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Portable Document Format (PDF-AAM) Accessibility APIs Mapping (AAM) Task Force of the ARIA WG
+permalink: /about/groups/task-forces/pdf-aam/
+ref: /about/groups/task-forces/pdf-aam/
+lang: en
+github:
+  label: wai-groups
+---
+
+{::nomarkdown}
+{% include box.html type="start" title="Summary" class="" %}
+{:/}
+
+The Portable Document Format Accessibility APIs Mapping (PDF-AAM) Task Force is a task force of the [Accessible Rich Internet Applications Working Group](/about/groups/ariawg/).
+
+Some information on this page is also shown on, and may be more current in, the [PDF-AAM page](https://www.w3.org/groups/tf/pdf-aam/).
```

Comment:
@tamsinewing555 please edit or delete this as appropriate for the current and future WG/TF page updates. thanks.